cookbooks/cdo-apps/recipes/default.rb

```diff
@@ -45,13 +45,7 @@
   cmake
 )
 
-#multipackage
-
-include_recipe 'cdo-mysql::client'
-# Install local mysql server unless an external db url is provided.
-unless node['cdo-secrets'] && node['cdo-secrets']['db_writer']
-  include_recipe 'cdo-mysql::server'
-end
+include_recipe 'cdo-mysql'
```

Comment:
Moved this bit of logic into `cdo-mysql` recipe to clean this recipe up slightly.